cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/TPFSSinkConfig.java

```diff
@@ -33,8 +33,13 @@
   @Nullable
   protected String basePath;
 
-  public TPFSSinkConfig(String name, @Nullable String basePath) {
+  @Description(TimePartitionedFileSetSink.PATH_FORMAT_DESC)
```

Comment:
etl plugins are also documented at cdap-docs/application-templates/source/etl/templates/sinks/batch/ (this is what the UI links to when you hit the info button for a plugin).  If we are adding